lib/mandrill_dm/message.rb

```diff
@@ -181,6 +192,9 @@ def combine_extra_header_fields # rubocop:disable MethodLength
         X-MC-TrackingDomain
         X-MC-URLStripQS
         X-MC-ViewContentLink
+        X-MC-Template
+        X-MC-AutoText
+        X-MC-MergeVars
```

Comment:
@genaromadrid the existing headers are in :abc: order, so new header fields should be added alphabetically.